index.html

```diff
@@ -228,11 +228,11 @@ <h4> <span class="icon"></span>Skills</h4>
        <section class="contact">
           <h3> I'm always interested in hearing about new projects,
               so if you'd like to chat please get in touch.</h3>
-              <form class="contact-form">
-                 <input type="text" placeholder="Full name" class="input">
-                 <input type="text" placeholder="Email address" class="input">
-                 <textarea  cols="30" rows="10" placeholder="Enter text here"></textarea>
-                 <input type="submit" value="Get in touch" class="btn">
+              <form class="contact-form" action="https://formspree.io/f/meqnpglr" method="post">
+                 <input type="text" placeholder="Full name" class="input" maxlength="30" required>
+                 <input type="email" placeholder="Email address" class="input" required>
+                 <textarea  cols="30" rows="10" placeholder="Enter text here" maxlength ="500" required></textarea>
+                  <button type="submit" class="btn">Get in touch</button>
```

Comment:
- [ ] It would be nice if you add the `name` property on all the input fields of the form element. Currently, the name, email, and message fields do not have the `name="something"` attribute. It is a must for them to have it in order for FormSpree to work properly. Kindly add that on all fields with their respective names passed in each field. 👍 
  - Hint: `<input name="email" id="email" type="email">`
  
**Below is the error I get when submitting the form**
![image](https://user-images.githubusercontent.com/50941074/165393177-62de7356-2e41-457e-acc9-dfc07ef590f8.png)

 
- [ ] Kindly adjust the styles of the form fields to take the full width of the screen. 

**The Figma Design**

![image](https://user-images.githubusercontent.com/50941074/165392867-aaa09a98-c3b1-4eb0-8ea4-3b5714e60abd.png)

**Your Design**

![image](https://user-images.githubusercontent.com/50941074/165393026-b93d58bf-ba77-495a-ae3d-835c3d2f3679.png)